### ISSUE TITLE
Combined username and password lists

### ISSUE THREAD

**Issue by glynx** [evilsocket/legba#14]
I commonly use lists containing username and password combinations to enumerate both default logins and shared passwords across multiple systems. In my experience a option to pass username:password pairs as a wordlist would be great.

Most tools and public lists use `:` as separator and i cannot recall ever seeing a username containing `:` so this may be a the best approach.

**Comment by evilsocket** [comment]
this sounds useful and doable ... one question, say you have this file:

```
user1:pass1
user2:pass2
```

should the feature only test these combinations, or also test:

```
user1:pass1
user2:pass2
user1:pass2
user2:pass1
```

?

**Comment by glynx** [comment]
Thanks for the quick response. I would only test the exact combinations like user1:pass1 and so on. If the user wants more permutations he can also generate such a list using other tools or just use separate user and password lists.

**Comment by evilsocket** [comment]
makes sense!

**Comment by evilsocket** [comment]
released as part of 0.4.0 https://github.com/evilsocket/legba/wiki/Usage-and-Main-Options#predefined-combinations
